tests/common/helpers/tacacs/tacacs_helper.py

```diff
@@ -341,7 +341,7 @@ def ssh_remote_run(localhost, remote_ip, username, password, cmd):
     return res
 
 
-def ssh_remote_run_retry(localhost, dutip, ptfhost, user, password, command, retry_count=3):
+def ssh_remote_run_retry(localhost, dutip, ptfhost, user, password, command, retry_count=6):
```

Comment:
@liuh-80  How much does the success rate improve after increasing the number of retries?

Reply:
Not sure, the issue is because tacplus server not stable, there are some bugs in that software. the only thing we can do is retry.